src/components/InformationDialog/InformationDialog.js

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect } from 'react';
```

Comment:
we could apply the const style approach we are using in the other files on this one as well.
instead adding as text, we would create an object with the props.
i think it gets better.
wyo?

Reply:
I think this is awesome!
This would work similarly to the Constants approach, just like used in `Fallback.js`:
![image](https://user-images.githubusercontent.com/13629975/92964326-4a540900-f44a-11ea-84de-cc9f1aa20feb.png)

I am going to change this 😉